pkgs/development/libraries/clipper2/0001-fix-pc-paths.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/Clipper2.pc.cmakein b/Clipper2.pc.cmakein
+index 5632c93..0a5d6ba 100644
+--- a/Clipper2.pc.cmakein
```

Comment:
dunno if I commented this before, but
1. It should use [GNUInstallDirs](https://cmake.org/cmake/help/v3.27/module/GNUInstallDirs.html) module.
2. It should be upstreamed

Reply:
I agree, I'll check if the upstream maintainers are good with it.